styles.css

```diff
@@ -884,6 +921,91 @@ i.fa-instagram {
         transform: scale(0);
     }
 }
+
+@media all and (max-width: 1440px) {
+    .testim-headline{
+        font-size: 2.25rem;
+    }
+    .testim .cont div h3{
+        font-size: 3rem;
+    }
+    .testim .cont div p{
+        font-size: 2rem;
+    }
+    .recommendation-container{
+        height: 100vh;
+    }
+}
+/*
+@media all and (max-width: 1425px) {
+    .testim .cont div p{
+        font-size: 2rem;
+    }
+}
+*/
+
+@media all and (max-width: 1024px) {
+    .testim-headline{
+        font-size: 2rem;
+    }
+    /*
+    .recommendation-container{
+        height: 60vh;
+    }
+    */
+
+    .testim .cont div p{
+        font-size: 1.70rem;
+    }
+    .testim .cont div h3{
+        font-size: 2rem;
+    }
+
+}
+@media all and (max-width: 768px) {
+    .testim .cont div p{
+        font-size: 1.50rem;
+    }
+    .testim-headline{
+        font-size: 1.125rem;
+    }
+    /*
+    .recommendation-container{
+        height: 90vh;
+    }
+    */
+}
+
+@media all and (min-width: 320px) and (max-width: 545px) {
+    .testim .cont div p{
+        font-size: 1rem;
+    }
+}
+
+@media all and (max-width: 320px) {
+    .testim .cont div p{
+        font-size: 0.875rem;
+    }
+    .testim-headline{
+        font-size: 1rem;
+    }
+    .logos {
+        padding-top: 0rem;
+    }
+    .testim .cont div h3 {
+        font-size: 1rem;
+    }
+    .testim .wrap {
+        padding: 30px 15px;
+    }
+    .recommendation-container{
+        height: 100vh;
```

Comment:
I would reduce a bit the vh - make it 70vh? because there is quite a big gap between the recommendation container and the logos.